test/progress-bar-test.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-console */
+
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from 'chai';
+
+import { ProgressBar } from '../src';
+
+describe('<ProgressBar/>', function() {
+  it('Should have a default class of primary ithout type property', function() {
+    const component = shallow(<ProgressBar percentage={0}/>);
+    expect(component.find('div.progress > div').hasClass('progress-bar-primary')).to.equal(true);
+  });
+
+  it('Should have a class of success', function() {
+    const component = shallow(<ProgressBar percentage={0} type='success'/>);
+    expect(component.find('div.progress > div').hasClass('progress-bar-success')).to.equal(true);
+  });
+
+  it('Should have a class of warning', function() {
+    const component = shallow(<ProgressBar percentage={0} type='warning'/>);
+    expect(component.find('div.progress > div').hasClass('progress-bar-warning')).to.equal(true);
+  });
+
+  it('Should have a class of danger', function() {
+    const component = shallow(<ProgressBar percentage={0} type='danger'/>);
+    expect(component.find('div.progress > div').hasClass('progress-bar-danger')).to.equal(true);
+  });
+
+  it('By default should not be animated', function() {
+    const component = shallow(<ProgressBar percentage={0}/>);
+    expect(component.find('div.progress > div').hasClass('animated')).to.equal(false);
+  });
+
+  it('Should have a default label of empty string', function() {
```

Comment:
Maybe we should change the text to something like `Should not display a label`, currently the test sounds like it should show a label html tag but have nothing inside it